Lib/_pyrepl/utils.py

```diff
@@ -208,7 +208,9 @@ def gen_colors_from_token_stream(
                 ):
                     span = Span.from_token(token, line_lengths)
                     yield ColorSpan(span, "soft_keyword")
-                elif token.string in BUILTINS:
+                elif (
+                    token.string in BUILTINS
+                    and not (prev_token and prev_token.exact_type == T.DOT)):
```

Comment:
```suggestion
                elif (
                    token.string in BUILTINS
                    and not (prev_token and prev_token.exact_type == T.DOT)
                ):
```

Just to match the style of the other ifs.

Reply:
of course

Reply:
Done